engine/apps/schedules/tasks/refresh_ical_files.py

```diff
@@ -41,22 +41,29 @@ def refresh_ical_file(schedule_pk):
         notify_ical_schedule_shift.apply_async((schedule.pk,))
 
     run_task_primary = False
-    if schedule.cached_ical_file_primary is not None:
-        if schedule.prev_ical_file_primary is None:
+    if bool(schedule.cached_ical_file_primary):
```

Comment:
Why to convert to bool? Can't we check if schedule.cached_ical_file_primary, which will handle None end ""?

Reply:
I think it was more explicit, but switched back to just check for cached_ical_file_primary